### ISSUE TITLE
No module named audiosr.__main__; 'audiosr' is a package and cannot be directly executed

### ISSUE THREAD

**Issue by Matteoki** [haoheliu/versatile_audio_super_resolution#1]
Windows 10
I created two different conda evironments to use this code on:
Python 3.10.13
Python 3.9.18

The installing procedure seems to do fine, it installs AudioSR successfully
![cmd_GBIE5P7hez](https://github.com/haoheliu/versatile_audio_super_resolution/assets/66631504/cba2b8d4-658e-47d3-acf9-cd6efd605956)

However when I try to run `Audiosr` or `Python -m Audiosr` (on the same shell) this error pops up
`No module named audiosr.__main__; 'audiosr' is a package and cannot be directly executed`

**Comment by hhlcorpusant** [comment]
Hi the following should work now:
```shell
pip3 install audiosr=0.0.2
```

Still I recommand to install with the following command to keep your tool up-to-date.
```shell
pip3 install git+https://github.com/haoheliu/versatile_audio_super_resolution

```

**Comment by madwurmz** [comment]
you recommend one command here, but the other command in the other issue, I'm confused.  🙊 
the potential for this tool seems very promising, maybe have it incorporated into other audio-webui tools?

**Comment by rikabi89** [comment]
I have the same error with audiosr==0.0.2 installed 
![image](https://github.com/haoheliu/versatile_audio_super_resolution/assets/85737703/f1d32a02-189d-4291-b3ca-e7e100962ae3)
![image](https://github.com/haoheliu/versatile_audio_super_resolution/assets/85737703/be72aae0-36a8-4b9a-a561-afced520aeb8)

**Comment by rikabi89** [comment]
I managed to work by running the script from this dir : `.\anaconda3\envs\audiosr\Scripts` `python audiosr  -i .\versatile_audio_super_resolution\example\speech.wav`
Make sure to install pytorch with CUDA otherwise it will just on on CPU. It creates an output but I can't tell its working as it doesn't sound much better.

**Comment by ubranch** [comment]
same

**Comment by falseywinchnet** [comment]
send boobs

import os
import torch
import logging
from audiosr import super_resolution, build_model, save_wave, get_time, read_list
import argparse

os.environ["TOKENIZERS_PARALLELISM"] = "true"
matplotlib_logger = logging.getLogger('matplotlib')
matplotlib_logger.setLevel(logging.WARNING)
torch.set_float32_matmul_precision("high")
audiosr = build_model(model_name="speech", device="auto")

waveform = super_resolution(
        audiosr,
        "C:\\pathtofile\\./filename.wav", #use a sampling rate less than 42! recommend 6k mono. use a file less than 10s long!.
        seed=42,
        guidance_scale=3.5,
        ddim_steps=50,
        latent_t_per_second=12.8
    )

save_wave(waveform, "C:\\pathtofile\\./", name="test", samplerate=48000) #note: do not put .wav at end of file name, it will do this automatically and grumble if you do it
